source/components/_header.erb

```diff
@@ -34,3 +34,11 @@
     </span>
   </a>
 </header>
+<div class="govuk-c-phase-banner">
```

Comment:
I'd suggest this go in its own component file, rather than adding it to the header _component_ file.